releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.4/1.2.4 - April 27, 2020
+
+* Add ability to modify client connection to App Configuration and Key Vault. Clients can be provided via ```ConfigurationClientBuilderSetup``` and ```SecretClientBuilderSetup```.
```

Comment:
>```ConfigurationClientBuilderSetup``` and ```SecretClientBuilderSetup``` [](start = 106, length = 72)

We can be more explicit that users can implement these interfaces to configure the default client.

Reply:
Yes, we need to provide more details than this. We can update the document and add a link in the release notes.